channels/ae.m3u

```diff
@@ -17,9 +17,9 @@ https://admdn3.cdn.mangomolo.com/adsports3/smil:adsports3.stream.smil/playlist.m
 https://admdn4ta.cdn.mgmlcdn.com/adsports4/smil:adsports4.stream.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="AjmanHD.ae" tvg-name="Ajman HD" tvg-country="AE" tvg-language="Arabic" tvg-logo="" group-title="",Ajman HD
 http://149.100.19.226:8001/play/AJMAN_HD
-#EXTINF:-1 tvg-id="AlAan.ae" tvg-name="Al Aan" tvg-country="AE" tvg-language="Arabic" tvg-logo="https://www.alaan.tv/wp-content/themes/alaan-child-tv/assets/img/logo.png" group-title="",Al Aan (720p) [Not 24/7]
+#EXTINF:-1 tvg-id="AlAan.ae" tvg-name="Al Aan" tvg-country="AE" tvg-language="Arabic" tvg-logo="https://www.alaan.tv/wp-content/themes/alaan-child-tv/assets/img/logo.png" group-title="Kids",Al Aan (720p) [Not 24/7]
 https://iptv--iptv.repl.co/streamlink?url=https://www.dailymotion.com/video/x74wje5
-#EXTINF:-1 tvg-id="AlAan.ae" tvg-name="Al Aan" tvg-country="AE" tvg-language="Arabic" tvg-logo="https://www.alaan.tv/wp-content/themes/alaan-child-tv/assets/img/logo.png" group-title="",Al Aan (720p)
+#EXTINF:-1 tvg-id="AlAan.ae" tvg-name="Al Aan" tvg-country="AE" tvg-language="Arabic" tvg-logo="https://www.alaan.tv/wp-content/themes/alaan-child-tv/assets/img/logo.png" group-title="Kids",Al Aan (720p)
```

Comment:
![Uploading image.png…]()
Don't assume genres by momentary programming

Reply:
Okay. I'll research some more. Those speakers can go for hours.